hack/ci/e2e-conformance.sh

```diff
@@ -253,9 +269,15 @@ build() {
 
   cd ${REPO_ROOT}
   echo "Build Docker Images"
-  make modules docker-build
+  make modules hack/tools/bin/clusterctl hack/tools/bin/kustomize hack/tools/bin/envsubst docker-build
 }
 
+# CAPI/CAPO versions
+# See: https://console.cloud.google.com/storage/browser/artifacts.k8s-staging-cluster-api.appspot.com/components?project=k8s-staging-cluster-api
+E2E_CAPI_DOWNLOAD_VERSION=nightly_master_20210305
```

Comment:
how can we make this up to date?

Reply:
We have to keep using the nightly versions until there is a first v1alpha4 release. (until then we use the nightly version of the day of our go.mod dependency to cluster-api)